src/ui-test/tests/codelens.test.ts

```diff
@@ -73,7 +74,11 @@ describe('JBang commands execution through command codelens', function () {
     });
 
     it(`Execute command 'apache.camel.debug.jbang' with codelens '${variables.CAMEL_DEBUG_CODELENS}'`, async function () {
-        await (await findCodelens(variables.CAMEL_DEBUG_CODELENS)).click();
+        if (process.env.CAMEL_VERSION !== undefined && isCamelVersionProductized(process.env.CAMEL_VERSION)){
```

Comment:
this statement can be simplified if you move `process.env.CAMEL_VERSION !== undefined ` directly into `isCamelVersionProductized`